onmt/modules/Conv2Conv.py

```diff
@@ -0,0 +1,106 @@
+"""
+Implementation of "Convolutional Sequence to Sequence Learning"
+"""
+import torch
+import torch.nn as nn
+import torch.nn.init as init
+import onmt.modules
+from onmt.modules.WeightNorm import WN_Conv2d
+import torch.nn.functional as F
+from torch.autograd import Variable
+
+scale_weight = 0.5 ** 0.5
+
+
+def shape_transform(x):
```

Comment:
What does this function do?

Reply:
Just trans the size of the tensor to fit the input of the conv layer. I have added a comment.